doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,3 +1,8 @@
 # Release X.Y (202?-??-??)
 
-* Allow explicitly selecting outputs with *store derivations* installable syntax too.
+* `<nix/fetchurl.nix>` now accepts an additional argument `impure` which
+  defaults to `false`.  If it is set to `true`, the `hash` and `sha256`
+  arguments will be ignored and the resulting derivation will have
```

Comment:
wouldn't it make more sense to make it an error to specify these arguments, or would that break when combined with indirect usage in nixpkgs or such?

Reply:
That is unrelated to this PR. I happen to agree but please bring up elsewhere.

Reply:
this stems from the merging of `master` right?